besu/src/main/java/org/hyperledger/besu/cli/subcommands/storage/RocksDbHelper.java

```diff
@@ -66,36 +67,213 @@ static void forEachColumnFamily(
     }
   }
 
-  static void printUsageForColumnFamily(
+  static void printStatsForColumnFamily(
       final RocksDB rocksdb, final ColumnFamilyHandle cfHandle, final PrintWriter out)
-      throws RocksDBException, NumberFormatException {
+      throws RocksDBException {
     final String size = rocksdb.getProperty(cfHandle, "rocksdb.estimate-live-data-size");
     final String numberOfKeys = rocksdb.getProperty(cfHandle, "rocksdb.estimate-num-keys");
-    boolean emptyColumnFamily = false;
-    if (!size.isBlank() && !numberOfKeys.isBlank()) {
+    final long sizeLong = Long.parseLong(size);
+    final long numberOfKeysLong = Long.parseLong(numberOfKeys);
+    if (!size.isBlank()
+        && !numberOfKeys.isBlank()
+        && isPopulatedColumnFamily(sizeLong, numberOfKeysLong)) {
+      out.println("=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=");
+      out.println("Column Family: " + getNameById(cfHandle.getName()));
+
+      final String prefix = "rocksdb.";
+      final String num_files_at_level_prefix = "num-files-at-level";
```

Comment:
I think this metric doesn't exist, you need to add the level number, like explain [here](https://github.com/facebook/rocksdb/blob/de1e3ff6eab9bc562ef262b1a49ec002b7bbbaef/include/rocksdb/db.h#L993)

Reply:
removed